app/controllers/synonyms/exports_controller.rb

```diff
@@ -2,7 +2,7 @@ module Synonyms
   class ExportsController < AuthenticatedController
     def create
       export_service = SearchReference::ExportAllService.new
-      filename = "all-synonyms-#{Time.zone.now.to_i}.csv"
+      filename = "all-synonyms-#{Time.zone.now.iso8601}.csv"
```

Comment:
TODO: There are no tests at all of this controller, yet. I have tested this manually in my local ui :scream_cat: